.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
Heads up we may also need to go into settings to properly enable Dependabot ([source](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#enabling-dependabot-version-updates))

Reply:
sgtm! I actually don't want it to work at the moment, so I think I'll just throw this file into the repo for future use whenever we enable it